packages/gatsby-remark-katex/src/index.js

```diff
@@ -1,22 +1,26 @@
 const visit = require(`unist-util-visit`)
```

Comment:
would it be okay to remove usage of `const parseHtml` and `const renderedKatexString` and just keep the previous form? I understand you want to remove duplicate code, but the previous form is easier to understand.

```js
const visit = require(`unist-util-visit`)
const unified = require(`unified`)
const parse = require(`rehype-parse`)
const katex = require(`katex`)
const remarkMath = require(`remark-math`)

module.exports = ({ markdownAST }, pluginOptions = {}) => {
  visit(markdownAST, `inlineMath`, (node) => {
    node.data.hChildren = unified()
      .use(parse, { fragment: true, position: false })
      .parse(
        katex.renderToString(node.value, {
          displayMode: false,
          ...pluginOptions,
        })
      ).children
  })

  visit(markdownAST, `math`, (node) => {
    node.data.hChildren = unified()
      .use(parse, { fragment: true, position: false })
      .parse(
        katex.renderToString(node.value, {
          displayMode: true,
          ...pluginOptions,
        })
      ).children
  })
}

module.exports.setParserPlugins = () => [remarkMath]

```